com.getstation.Station.yaml

```diff
@@ -0,0 +1,50 @@
+id: com.getstation.Station
+base: org.electronjs.Electron2.BaseApp
+base-version: '24.08'
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: station
+separate-locales: false
+rename-desktop-file: station-desktop-app.desktop
+rename-icon: station-desktop-app
+finish-args:
+  - --share=ipc
+  - --device=dri
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --env=ELECTRON_TRASH=gio
+modules:
+  - name: station-desktop
+    buildsystem: simple
+    build-commands:
+      - ar x Station-*.deb
```

Comment:
```suggestion
      - ar x Station.deb
```

Reply:
If you don't mind, I prefer to keep it

Reply:
if you don't mind, reviewers ask you to do so.